src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+  const declarations = styles.split(';');
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':');
+
+    if (property && value) {
```

Comment:
The condition `if (property && value)` checks for the existence of both `property` and `value`, but it does not account for cases where `property` or `value` might be whitespace. Consider trimming `property` and `value` before this check.